private/extensions/maven.bzl

```diff
@@ -338,6 +340,8 @@ def _maven_impl(mctx):
             if mod.is_root:
                 repo["repin_instructions"] = install.repin_instructions
 
+            repo["additional_coursier_options"] = repo.get("additional_coursier_options", []) + getattr(install, "additional_coursier_options", [])
```

Comment:
I'm not very sure I'm doing the right thing, so please let me know if anything missed.

Reply:
This is perfect